util/get_vulns.js

```diff
@@ -0,0 +1,18 @@
+import fetch from 'node-fetch';
+const snykApi = 'http://snyk-widget.herokuapp.com';
+const key = 'VRHr?YP_Gb7d9LFh$ugktZHUe+gGBp#LGG3*bDLt';
```

Comment:
Is this key safe to be public?

Reply:
Yes, the data is open. However, if we see someone is overusing it we can rate limit. That's why we hand out separate keys